### DISCUSSION TITLE
Question: list all applications and application windows

### DISCUSSION THREAD

**Discussion by jakub-m** [progrium/darwinkit#95] [General]
Hi!

First, thanks for your awesome work on macdriver!

Question: is it possible to list active applications and their windows, or just all the active windows? I looked through macdriver and didn't find anything like "list applications" or "list windows". I checked StackOverflow, the only suggestion I found is "accessibility API" but I didn't find such binding in macdriver.

Thanks!

**Comment by progrium** [comment]
If you can find the Apple docs on how to do this with Objective-C or Swift, I'm sure somebody can show you how to do it with macdriver in Go.

**Comment by progrium** [comment]
Closing for now but if you find the Apple docs on how to do it, I can help translate the Obj-C or Swift to Go.
